### PR TITLE
Added --no-redact parameter

### PR DESCRIPTION
**Pull request by jszroberto**
I have added the option to configure if you want a redacted output or not. Can you review it? @vito

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ the UUID returned by the targeted director.
 
 * `cleanup`: *Optional* An boolean that specifies if a bosh cleanup should be
   run before deployment. Defaults to false.
-
+* `no-redact`: *Optional* Removes redacted from Bosh output. Defaults to false.
```
Comment (vito) [inline README.md]: This should use an underscore rather than a hyphen.

**Comment by vito** [comment]
Left an inline comment. I think a test could also be added.

**Comment by jszroberto** [comment]
@vito would you like us to do it? or are you planning to do it yourself?

**Comment by vito** [comment]
@jszroberto makes more sense for you to do it in principle :)

**Comment by jszroberto** [comment]
@vito can you review now?

File: lib/bosh_deployment_resource/bosh.rb [hunk 2]
```diff
@@ -10,6 +10,15 @@ def initialize(target, ca_cert, auth, command_runner=CommandRunner.new)
       @ca_cert = ca_cert
       @auth = auth
       @command_runner = command_runner
+      @no_redact = false
+    end
+
+    def enable_redact()
```
Comment (vito) [inline lib/bosh_deployment_resource/bosh.rb]: I don't think this should be stateful. If instead it's an argument to `deploy` these aren't necessary.

File: lib/bosh_deployment_resource/bosh.rb [hunk 3]
```diff
@@ -21,7 +30,11 @@ def upload_release(path)
     end
 
     def deploy(manifest_path)
-      bosh("-d #{manifest_path} deploy")
+      if @no_redact
```
Comment (vito) [inline lib/bosh_deployment_resource/bosh.rb]: Would be simpler if this were just an argument.

File: lib/bosh_deployment_resource/out_command.rb [hunk 4]
```diff
@@ -13,7 +13,7 @@ def run(working_dir, request)
       validate! request
 
       bosh.cleanup if request.fetch("params")["cleanup"].equal? true
-
+      bosh.enable_redact if request.fetch("params")["no_redact"]
```
Comment (vito) [inline lib/bosh_deployment_resource/out_command.rb]: This can be passed to `deploy` below instead.

Comment (jszroberto) [inline lib/bosh_deployment_resource/out_command.rb]: @vito done, can you check now?

**Comment by vito** [comment]
Looks good, thanks!

**Pull request merged**